### DISCUSSION TITLE
Systematic difficulties of Fort Canning

### DISCUSSION THREAD

**Discussion by blttgrst** [DeFiCh/app#1090] [Ideas]
Hey there, 

after the launch of Fort Canning and the possibility to create vault, I made some thoughts about my investing strategy on my own, despite the vids on youtube, etc. I thought, what happens, if more people wants to sell DFI for DUSD to get into stocks? Especially the cake-users, which can´t do anything else so far. 
My first answer was, that they would "dump" the DFI-DUSD-Pool and take some disadvantage of an non-equal pool.
How can the other side work against that sell-pressure and what happens, when they do. As Daniel Zirkel mentioned in his tutorial, arbitrage is only working over time, so you have to wait. But how long? Would I buy even more "cheap" DFI to wait for it? I won´t could go back than.
After it turned out that exactly his case happened just in the first day, I was sure, that this is a problem in the system. 

Here is what I see:
Why is it possible to sell "crypto"-DFI to stock-token via DUSD, when there is the convenient way of a vault, where you also can long/neutral/short the stocks? It doesn´t matter, where you sell DFI. You are selling it. A gateway to stocks only through vaults, would create the stocktoken and DUSD and leave it closed in the system. It can´t be traded outside anyway. The other way would be, that every stock-token could be traded in a e.g. DFI-dTLSA-Pool. So when you arbitrage the DFI-DUSD-Pool, you can than sell it again for a stock-token and the circle would be closed. 

So there are two options:
1) Closed system in the blockchain itself by getting stock-token only through by minting DUSD or the stock-token itself.
2) Opening the system by starting for every stock-token an DFI- and an DUSD-Pool.

I think going back to Option 1 would be very difficult now, but in my point of view would it be the best. 
In option 2 we have to split the rewards (of course) for the corresponding DFI-Pools.  This option would also make it easier to get stock-token without DUSD, but of course the liquidity-miners have to deal with a bigger impermanent loss. Therefore the APR would be higher probably balancing the risk. 

What do you think about? Let me know.

**Comment by LumpiesRevenge** [comment]
Your proposal No.1 would definitely improve the current imbalance between DEX and Oracle prices of the dTokens.
But it will make "decentralized stock trading" impossible = it won´t attract any stock traders from the real world to the DeFiChain.
Because going the route via vault-only is:
1) a complex mechanism (nothing for a simple minded stock trader)
2) has additional risks (liquidation)
3) is very capital inefficient for the loan taker
4) is a very slow and (on its own) inefficient price harmonizing mechanism (no instant arbitrage possible, Oracle price changes every hour). If we want to attract stock traders the price difference must be constantly regulated by the market actors/algorithm close to 1-0,5% deviation.

So "road to 50" will never happen because the dToken ecosystem is not able to scale.

**Comment by LumpiesRevenge** [comment]
Concerning Option 2):

This wouldnt fix the dToken supply problem. It would only remove half the trading volume from the dUSD-DFI pool (so the inbalance will half) and distribute it between the newly created DFI-dToken Pools imo (and increase their inbalance accordingly).
